src/Microsoft.Tye.Hosting/TyeHost.cs

```diff
@@ -109,6 +109,7 @@ public async Task<IHost> StartAsync()
             try
             {
                 await _processor.StartAsync(_application);
+                _logger.LogInformation($"Application {_application.Name} started successfully with Pid: {Process.GetCurrentProcess().Id}");
```

Comment:
I don't know if it makes any significant difference, but would it be better to log this as early as possible to reduce the length of time required to scan the process output?

Reply:
It doesn't really wait for all the processes to start. The heavy wait processors like `DockerRunner` don't wait on the containers to be up, so we get this output relatively quickly.

The current information message gives the impression that all the processes are started, but I think I can modify the information message a little bit to reflect that not all the processes are started, but Tye Process is started.

Reply:
Ok, that's fine.  Since the intent is for tooling to read this message, maybe something that's a little more "regular" or "extractable"?  I know there's a balance with wanting readibility, but maybe something more terse like:

```
Application: {_application.Name}
PID: {Process.GetCurrentProcess().Id}
```